applications/polkakeeper.md

```diff
@@ -0,0 +1,302 @@
+
+# Contact
+* Contact Name: LOH ZHENG RONG
+* Contact Email: zr@rampdefi.com
+* Website: https://polkakeeper.com
+
+# Legal Structure
+* Registered Address: 14 ROBINSON ROAD #08-01A FAR EAST FINANCE BUILDING S(048545), SINGAPORE
+* Registered Legal Entity: Ramp Protocol Pte Ltd
+* Country of registration: Singapore
+
+# Ecosystem Fit
+**Comparable projects**
+* Keep3r.network: https://keep3r.network/ (Ethereum)
+* KeeperDao: https://keeperdao.com/ (Ethereum)
+* Web3 Guardian : https://github.com/open-web3-stack/guardian (Ethereum)
+
+# Milestones:
+### Overview
+* **Total Estimated Duration:** 3 Months / POC
+* **Full-time equivalent (FTE):**  2.75
+* **Total Costs:** 0.55 BTC
+
+### Technical Milestones
+
+#### Milestone 1: Preparation (0.10 BTC)
+| Deliverable   | Description                                                                                |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| License       | All code will be open-sourced under the MIT license and shared in github or gitlab. We welcome contribution and reuse.|
+| Source code   | Public source code repository containing a Rust codebase based on the Substrate Node Template would be set up. Programming language will be Rust, using the Substrate Framework.|
+| Source code   | Provide both inline documentation of the code and a guide that explains how to integrate the Vault, Liquidator and LiquidatorAdaptor pallets. |
+| Source code   | Write the adapters for a target platform. This release would include Vault and Liquidator pallet outlines, to validate the approach and structure. |
+| UI / Functionality | Design specifications (Stated in UI Development Segment: User stories, mockups) for the user-facing functionality, which would be used in the next phase to build the React UI in Milestone 2.|
+| Community     | Publish a Medium Article that announces all deliveries and a step by step tutorial for the developer community once evaluation is passed |
+| Documentation | Initial public outline of the documentation |
+
+#### Milestone 2: Alpha Release (0.25 BTC)
+| Deliverable | Description                                                                                                                            |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| License     | All code will be open-sourced under the MIT license and shared in github or gitlab. We welcome contribution and reuse.                 |
+| Source code| Vault, Liquidator and LiquidatorAdaptor pallets all implemented with basic (<30%) unit test coverage.|
```

Comment:
Could you outline the difference in functionality between Milestone 2 and 3, apart from unit test coverage? Will the solution be feature-complete by the end of M2?

Reply:
@mmagician besides the unit test coverage, the other key difference between milestone 2 and 3 is that milestone 2 would be feature complete for internal use but would not yet be connected to a real chain for general use whereas milestone 3 would be connected to a real chain and ready for general public use.